helm/superset/Chart.yaml

```diff
@@ -29,7 +29,7 @@ maintainers:
   - name: craig-rueda
     email: craig@craigrueda.com
     url: https://github.com/craig-rueda
-version: 0.8.4
+version: 0.8.5
```

Comment:
Pls bump this to `0.8.6` as this PR is likely to collide: #22854

Reply:
Done, thanks.